JPS_VIRTUALSENSOR/AermodAgent/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.8"
 
 services:
   aermod-agent:
-    image: ghcr.io/cambridge-cares/aermod-agent:1.1.2-SNAPSHOT
+    image: ghcr.io/cambridge-cares/aermod-agent:1.1.2-1559-add-citieskg-namespace-as-derivation-input-SNAPSHOT
```

Comment:
remember to increment the number

Reply:
The number has been changed to 1.1.3 in the latest commit in this docker-compose.yml and also the startup.sh and aermod-agent.json file in the stack-manager directory.